### PR TITLE
Admin domain blocks - adding the new v4 admin API endpoints 

### PR DESCRIPTION
**Pull request by JacksonSamuel42**

### CODE REVIEW

File: src/repositories/admin/domain-blocks-repository.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import type { MastoConfig } from '../../config';
+import { version } from '../../decorators';
+import type { Admin } from '../../entities';
+import type { SeverityDomain } from '../../entities/admin/domain-blocks';
+import type { Http } from '../../http';
+
+export interface AdminDomainBlocksFetchParams {
```
Comment (neet) [inline src/repositories/admin/domain-blocks-repository.ts:7]: It's not for `fetch` method, so should be named like `AdminBlockDomainParams`

File: src/repositories/admin/domain-blocks-repository.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  }
+
+  /**
+   * Lift a block against a domain.
+   * @param id id of domain
+   * @return DomainBlocks
+   * @see https://docs.joinmastodon.org/methods/admin/
+   */
+  @version({ since: '2.9.1' })
+  delete(id: string): Promise<void> {
```
Comment (neet) [inline src/repositories/admin/domain-blocks-repository.ts:95]: I'd like to name it `remove` for consistency https://github.com/neet/masto.js/search?q=remove

**Review by neet — CHANGES_REQUESTED** [review]
- Masto.js internally converts property names from `snake_case` to `camelCase` to comply with JS conventions, so please update all `snake_cases` to annotate types properly
- It seems you forgot to add `export` to sec/entities/admin/index.ts. Otherwise you cannot `import from "masto"` https://github.com/neet/masto.js/blob/main/src/entities/admin/index.ts

**Comment by JacksonSamuel42** [comment]
@neet i will work in this currections

File: src/entities/admin/domain-blocks.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+export type SeverityDomain = 'silence' | 'suspend' | 'noop';
+
+export interface DomainBlocks {
```
[the remark this answers is not in the document]
Comment (JacksonSamuel42) [reply src/entities/admin/domain-blocks.ts:3]: i dont undestand the snack_case point i put all properties in camelCase it should be everything ok

[the remark this answers is not in the document]
Comment (JacksonSamuel42) [reply src/entities/admin/domain-blocks.ts:3]: @neet i get the point kk

[the remark this answers is not in the document]
Comment (JacksonSamuel42) [reply src/entities/admin/domain-blocks.ts:3]: nevermind

**Pull request closed without merging**